docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -933,7 +940,9 @@ A query clause that retrieves a subset of data from the source index. See
 end::source-query-transforms[]
 
 tag::source-runtime-mappings-transforms[]
-Definitions of search-time runtime fields that can be used by the transform.
+Definitions of search-time runtime fields that can be used by the transform. If 
+you use <<modules-cross-cluster-search,{ccs}>>, the version of the remote 
+cluster must be newer than 7.12.
```

Comment:
Is it 7.12 and higher? Or just 7.13?  If the former, I suggest:
```suggestion
cluster must be 7.12 or later.
```

Reply:
The issue says "the remote must be > 7.12", so I assume it has to be at least 7.13. @hendrikmuhs can you please confirm it?

Reply:
its 7.12 or later.

I think it sounds nicer if you say: For search runtime fields all data nodes, including remote nodes, must be 7.12 or later.